script/Arcueid/Day 10/10_00_ARC10_6.txt

```diff
@@ -1022,13 +1022,13 @@ The first one is that they must be wealthy.
 [sha:83380ef9ab7b6e8944a05c12a6d5112e9d61e885]{
 -- Page 732, Offset 26260.
 -- 魔術のような学んで修得する神秘ではなく、生まれついてその肉体が持ってしまう特異能力。
-I'm not talking about mysteries you can learn, like magecraft,
+I'm not talking about the Arcane which you can study, like magecraft,
  but supernatural powers that are present in the body from the moment it's born.
 }
 [sha:05fa1e158f5a48ceedc0d419bbb15519e7664805]{
 -- Page 732, Offset 26261.
 -- 一般に超能力者、鬼子と差別されるもの。
-In common terms, those people are referred to as Psychics, or %{i}Changelings%{/i}.
+In common terms, those people are discriminated against, and labeled as psychics, or changelings.
```

Comment:
This splices the 'common terms' a bit, since it injects the discrimination before the labels. Maybe reorg
> People tend to discriminate against those that have these powers, branding them 'psychics' or 'changelings'.